### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by roliverosc**
Hello @enen92 have translated your add-on to spanish. I hope it is to your liking.
Regards and thank you for this add-on has been very useful to me.

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -10,10 +10,12 @@
     <extension point="xbmc.service" library="main.py"/>
     <extension point="xbmc.addon.metadata">
         <summary lang="en_GB">Cast videos from the Youtube application to Kodi</summary>
-        <summary lang="pt_PT">Cast de vídeos a partir da aplicação móvel Youtube para o Kodi</summary>
```
Comment (enen92) [inline addon.xml:13]: @roliverosc please fix the indentation of the file

Comment (roliverosc) [reply addon.xml:13]: I did not the last time. But now it is fixed. Not?

Comment (enen92) [reply addon.xml:13]: doesn't seem to be :P

Comment (roliverosc) [reply addon.xml:13]: Fixed? https://github.com/roliverosc/script.tubecast/commit/0192e3918b5911d86afa38d4c637b004daee1af2

**Pull request merged**